rocket/rocket.py

```diff
@@ -54,49 +53,53 @@ def launch(
             project_location: str = ".",
             dbfs_path: Optional[str] = None,
             watch: bool = True,
-            glob_path: Optional[Union[str, List[str]]] = None
+            glob_path: Optional[Union[str, List[str]]] = None,
+            use_volumes: Optional[bool] = False,
+            dst_path: Optional[str] = None,
     ) -> None:
         """
         Entrypoint of the application, triggers a build and deploy
         :param project_location: path to project code, default: `"."`
-        :param dbfs_path: path where the wheel will be stored, ex: dbfs:/tmp/myteam/myproject
+        :param dbfs_path: path where the wheel will be stored, ex: dbfs:/tmp/myteam/myproject. Only support dbfs path.
         :param watch: Set to false if you don't want to automatically sync your files
         :param glob_path: glob string or list of strings for additional files to deploy, e.g. "*.json"
+        :param use_volumes: upload files to unity catalog volumes.
+        :param dst_path: Destination path to store the files. Support both dbfs:/ and /Volumes. Ideally, we should use dst_path and deprecate dbfs_path.
         :return:
         """
         if os.getenv("DATABRICKS_TOKEN") is None:
             raise Exception("DATABRICKS_TOKEN must be set for db-rocket to work")
-
-        if dbfs_path is not None and not dbfs_path.startswith("dbfs:/"):
-            raise Exception("`dbfs_path` must start with dbfs:/")
-
         try:
             execute_shell_command(f"databricks fs ls dbfs:/")
         except Exception as e:
             raise Exception(
                 f"Error accessing DBFS via databricks-cli. Please check if your databricks token is set and valid? Try to generate a new token and update existing one with `databricks configure --token`. Error details: {e}"
             )
 
-        if not dbfs_path:
-            dbfs_path = f"dbfs:/temp/{os.environ['USER']}"
+        if use_volumes:
```

Comment:
Maybe something like `if is_dbfs_path`

Reply:
If we did that, we'd need to choose some default behavior in case no path is provided (since `dst_path` is optional), so it would have to be something like:

`if dst_path is not None and is_dbfs_path(dst_path)`

And so you wouldn't be able to upload to dbfs without providing some path, which would be backwards incompatible.